features/step_definitions/steps.rb

```diff
@@ -0,0 +1,27 @@
+Given("I am on the landing page") do
+  visit 'welcome/index'
+end
+  
+  Given("I click on the {string} button") do |string|
+    click_on(string)
```

Comment:
Please make a choice regarding code style. If you use poetry mode (as you do below), please be consistent.